src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java

```diff
@@ -4,9 +4,7 @@
  *
  * This code is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation. Oracle designates this
- * particular file as subject to the "Classpath" exception as provided
- * by Oracle in the LICENSE file that accompanied this code.
+ * published by the Free Software Foundation.
```

Comment:
I don't understand this change. This file _needs_ the Classpath exception.

Reply:
Thanks Kevin. I have updated the PR.

Reply:
This is not an easy thing to see.